server/src/main/java/org/opentosca/toscana/plugins/cloudformation/visitor/CloudFormationNodeVisitor.java

```diff
@@ -229,67 +228,74 @@ private void handleOperation(Operation operation, String serverName, String conf
 
         //Add dependencies
         for (String dependency : operation.getDependencies()) {
-            try {
-                String cfnFileMode = "000644"; //TODO Check what mode is needed (only read?)
-                String cfnFileOwner = "root"; //TODO Check what Owner is needed
-                String cfnFileGroup = "root"; //TODO Check what Group is needed
-
-                // TODO: re-allow content-dumping instead of source
-                CFNFile cfnFile = new CFNFile(cfnFilePath + dependency)
-                    .setContent(cfnModule.getFileAccess().read(dependency))
-                    .setMode(cfnFileMode)
-                    .setOwner(cfnFileOwner)
-                    .setGroup(cfnFileGroup);
-
-                // Add file to install
-                cfnModule.getCFNInit(serverName)
-                    .getOrAddConfig(CONFIG_SETS, config)
-                    .putFile(cfnFile);
-            } catch (IOException e) {
-                logger.error("Problem with reading file " + dependency);
-                e.printStackTrace();
-            }
+            String cfnFileMode = "000644"; //TODO Check what mode is needed (only read?)
```

Comment:
Always fix values? Then you could make them `final static`

Reply:
Yes, for now, they are always fixed values, I changed them to final static accordingly.

In the future, this might be changed, see the TODOs and #406.